database/models/User.js

```diff
@@ -4,8 +4,12 @@ class User extends bookshelf.Model {
   get tableName() { return 'users' }
   get timestamps() { return true; }
 
-  users() {
-    return this.hasMany('Card');
+  created() {
+    return this.hasMany('Card', 'created_by');
+  }
+
+  assigned() {
```

Comment:
This should probably be named `cardsAssigned`. Because it's more about how this user can "have many" cards they've been assigned.